drivers/windows/mutex_windows.cpp

```diff
@@ -75,11 +75,13 @@ void MutexWindows::make_default() {
 	create_func = create_func_windows;
 }
 
-MutexWindows::MutexWindows() {
-
+MutexWindows::MutexWindows()
 #ifdef WINDOWS_USE_MUTEX
-	mutex = CreateMutex(NULL, FALSE, NULL);
-#else
+		:
```

Comment:
This one also looks weird, I'd prefer to keep the previous code.

Reply:
No problemo. Change undone.